### ISSUE TITLE
Authentication Error even though i am giving the right credentials

### ISSUE THREAD

**Issue by surabhi-shukla206** [swiftmailer/swiftmailer#223]
Hello  Team,

While sending the mail using swift mailer i am getting the following error :-

Fatal error: Uncaught exception 'Swift_TransportException' with message 'Failed to authenticate on SMTP server with username "nsiqa.2006@gmail.com" using 2 possible authenticators' in /var/www/vhosts/xyz.com/app/vendors/swift_mailer/classes/Swift/Transport/Esmtp/AuthHandler.php:179 Stack trace: #0 /var/www/vhosts/xyz.com/app/vendors/swift_mailer/classes/Swift/Transport/EsmtpTransport.php(252): Swift_Transport_Esmtp_AuthHandler->afterEhlo(Object(Swift_SmtpTransport)) #1 /var/www/vhosts/xyz.com/app/vendors/swift_mailer/classes/Swift/Transport/AbstractSmtpTransport.php(108): Swift_Transport_EsmtpTransport->_doHeloCommand() #2 /var/www/vhosts/xyz.com/app/vendors/swift_mailer/classes/Swift/Mailer.php(74): Swift_Transport_AbstractSmtpTransport->start() #3 /var/www/vhosts/xyz.com/app/controllers/components/swift_mailer.php(420): Swift_Mailer->send(Object(Swift_Message), Array) #4 /var/www/vhosts/xyz.com/app/app_controller.php(537): SwiftMailerComponent->send('default', 'Thanks for jo in /var/www/vhosts/xyz.com/app/vendors/swift_mailer/classes/Swift/Transport/Esmtp/AuthHandler.php on line 179

Details :- 
PHP Version - 5.3.13
Swift Mailer Verison - 2.30

Code:- 

```
           $this->SwiftMailer->smtpHost = 'ssl://smtp.gmail.com';
    $this->SwiftMailer->smtpPort = 465; 
    $this->SwiftMailer->sendAs  = 'html';
    $this->SwiftMailer->smtpTimeout   = 30;

    $this->SwiftMailer->smtpUsername  = 'nsiqa.2006@gmail.com';
    $this->SwiftMailer->smtpPassword  = 'nsiqa2006';

    $this->SwiftMailer->from = $from;
    $this->SwiftMailer->fromName = $replyTo;
    $this->SwiftMailer->to = $to;

    foreach($emailData as $name=>$value){
        $this->set($name, $value);
    }

    $status = $this->SwiftMailer->send($template, $subject);
    return $status;
```

The above error, i am getting while working on my live server, same code when i am using on my local server it is working very fine. Please suggest what could be the issue , why i am facing this error.

**Comment by tincho** [comment]
Hi, I just found this on the same Google results page that includes this answer:
https://stackoverflow.com/a/20656870/6732164

TL;DR is not a SwiftMailer but a Gmail security restriction 

 Just wanted to add this info here for the record. Best!
